helm/prefect-server/templates/apollo/deployment.yaml

```diff
@@ -61,7 +61,7 @@ spec:
             httpGet:
               path: /.well-known/apollo/server-health
               port: http
-          {{- with .Values.apollo.resources -}}
+          {{- with .Values.apollo.resources }}
```

Comment:
For my own knowledge, what are these changes for?  Seems to be removing a closing `-`?

Reply:
When `apollo.resources` is not defined, that whole block is empty as intended. When `apollo.resources` was defined, it was templated in but the `-` strips whitespace so we ended up with

```
              port: httpresources:
              limits: 
                cpu: 12
```
removing the trailing `-` doesn't strip the newline/indent before `resources:`

I'd like to write some python tests to ensure everything optional templates in correctly at some point.

Reply:
Ahh interesting, makes sense - thanks!